### PR TITLE
Feature/default shipping

### PR DESCRIPTION
**Pull request by arturmagalhaesjr**
#### What problem is this solving?

- The current logic for setting the default shipping address for a cost center is to use the first address in the array, which would be the original (oldest) address

- However, this is already causing some issues for HH because they need the ability to change which address is the default

#### How to test it?

It provided a default address toggle button to handle it.

Workspace:

- My Account:
[https://b2borg--sandboxusdev.myvtex.com/account](https://b2borg--sandboxusdev.myvtex.com/account)

- Admin:
[https://b2borg--sandboxusdev.myvtex.com/admin/b2b-organizations](https://b2borg--sandboxusdev.myvtex.com/admin/b2b-organizations)

#### Admin:
image

### My Account
image

### CODE REVIEW

File: react/admin/CostCenterDetails.tsx [hunk 1]
```diff
@@ -143,23 +148,54 @@ const CostCenterDetails: FunctionComponent = () => {
...
+        return item
+      })
+    )
+  }
+
   useEffect(() => {
+    if (data?.getCostCenterById?.addresses)
+      handleSetAddresses(data.getCostCenterById.addresses)
+
     if (addresses.length || !data?.getCostCenterById?.addresses?.length) return
```
Comment (polishq) [inline react/admin/CostCenterDetails.tsx:176]: I think `handleSetAddresses` should happen after the `if (address.length || ...` conditional. I think the current code is causing a bug. You can reproduce the bug like so:

- Go to `/admin/b2b-organizations/organizations`
- Click into an organization
- Click into a cost center
- Click the "back to organization" link
- Click into the same cost center again
- The list of addresses will be blank

Comment (arturmagalhaesjr) [reply react/admin/CostCenterDetails.tsx]: Yes, I got it too.
It's done

**Review by arturmagalhaesjr — COMMENTED** [review]
it's done

**Review by polishq — APPROVED** [review]
LGTM

**Pull request merged**